### ISSUE TITLE
My Android Tools app being updated??

### ISSUE THREAD

**Issue by jawz101** [android-police/apkmirror-public#88]
This app hads't been on the play store for a year.  Can we trust where these are coming from?  Where are these new versions coming from?  I tried reaching out to Wang Qi several months ago with no response so seeing new random versions seems fishy.

https://www.apkmirror.com/apk/wangqi/my-android-tools/my-android-tools-1-6-8-release/my-android-tools-1-6-8-android-apk-download/

**Comment by archon810** [comment]
Well, the signatures match, but beyond that I can't tell you. If the developer key got compromised and someone started releasing updates to this pretty obscure app, that's a little hard for us to determine and is beyond the scope of our verification process unless there's clear evidence of this, in which case we'd remove the updates immediately.

I ran the latest APKs through VirusTotal and there's either no or one flag by some random AV, very likely a false positive.

**Comment by jawz101** [comment]
Thanks.  It's actually a pretty powerful app, btw.  The only app I've found that lets you disable activities, broadcast receivers, services, & content providers... and generally see how complex some apps are.  Like Facebook and Amazon's apps are as elaborate as Google Play Services when you see how many custom services and receivers they make for themselves.  There's a reason it was removed from the Play Store :(
